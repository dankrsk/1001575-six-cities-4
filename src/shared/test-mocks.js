export const mock = {
  rentOffersCount: 3,
  offers: [
    {
      id: 1,
      type: `House`,
      title: `Beautiful & luxurious apartment at great location`,
      price: 100,
      rating: 1,
      isPremium: true,
      photo: `http://cdn.home-designing.com/wp-content/uploads/2013/11/1-Contemporary-architecture.jpg`,
      coordinates: [52.3909553943508, 4.929309666406198],
    },
    {
      id: 2,
      type: `Room`,
      title: `Wood and stone place`,
      price: 200,
      rating: 2,
      isPremium: false,
      photo: `https://avatars.mds.yandex.net/get-pdb/215709/4d8ee911-7746-43da-a5cd-b7f9d2f3f839/s1200?webp=false`,
      coordinates: [52.369553943508, 4.85309666406198],
    },
    {
      id: 3,
      type: `Appartment`,
      title: `Nice, cozy, warm big bed apartment`,
      price: 300,
      rating: 3,
      isPremium: true,
      photo: `https://avatars.mds.yandex.net/get-pdb/215709/4d8ee911-7746-43da-a5cd-b7f9d2f3f839/s1200?webp=false`,
      coordinates: [52.3909553943508, 4.85309666406198],
    },
  ]
};
