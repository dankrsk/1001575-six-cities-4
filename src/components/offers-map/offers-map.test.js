import React from 'react';
import renderer from 'react-test-renderer';
import OffersMap from './offers-map.jsx';
import {mock} from '../../shared/test-mocks.js';

describe(`Snapshots for Map`, () => {
  it(`Map component`, () => {
    const tree = renderer
      .create(<OffersMap
        offers={mock.offers}
        city={`Amsterdam`}
        activeCardId={1}
        isNearPlaces={true}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
