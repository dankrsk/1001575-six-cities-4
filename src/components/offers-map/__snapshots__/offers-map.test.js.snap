// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots for Map Map component should render container for map 1`] = `
<section
  className="cities__map map"
/>
`;
